src/components/missions.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getMission } from '../redux/missions/missionSlice';
+import Mission from './mission';
+import './styles/mission.css';
+
+export default function Missions() {
+  const { mission } = useSelector((state) => state.mission);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getMission());
+  }, [dispatch]);
```

Comment:
[Optional] The getMission action seems to be dispatched multiple times. It is required for the actions to be dispatched once. Kindly fix it as the previous reviewer seemed to have missed this.

The same applies to the rockets component. Check out the illustration below:

https://user-images.githubusercontent.com/20554583/232152663-587dd347-5b3c-4c5e-9aeb-d6e93e8fe289.mp4

Reply:
I never missed this everything was working fine when I reviewed this work the first time